### PR TITLE
Update to v 2.5.2:

### PR DESCRIPTION
**Pull request by akabanovs**
- bump up version and update sha256
 - skip for py<37
 - update script as 'python setup.py install' is deprecated
 - update dependencies
 - update the 'about' section

### CODE REVIEW

**Comment by akabanovs** [comment]
I was able to build using the original script:

python setup.py install  

but linter says this is deprecated, so I updated the recipe to use pip instead.  I hope this is correct?

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by ELundby45 — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,36 +1,39 @@
-{% set version = "2.5.1" %}
-{% set sha256 = "1d40df18e1c5a346faf44716573dfd531343936bf479cb05e9391dbe28b90779" %}
+{% set name = "jupyter_kernel_gateway" %}
+{% set version = "2.5.2" %}
 
 package:
   name: jupyter_kernel_gateway
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `  name: {{ name }}`

Comment (akabanovs) [reply recipe/meta.yaml]: Thanks

**Pull request merged**